src/collections/README.md

```diff
@@ -1,6 +0,0 @@
-# Collections Earnings (BETA)
```

Comment:
do we need to keep a simple README.md even if its empty

Reply:
Removal reverted